analysis/dataset_definition.py

```diff
@@ -41,25 +41,6 @@
 dataset.age = patients.age_on(start_date)
 dataset.age_greater_equal_65 = dataset.age >= 65
 
-# Check if a patient has a clinical code in the time period defined
-# above (selected_events) that are in the f2f_consultation codelist
-dataset.has_f2f_consultation = selected_events.where(
-    clinical_events.snomedct_code.is_in(f2f_consultation)
-).exists_for_patient()
-
-# Count number of f2f_consultation that a patient had
-# in the time period defined above (selected_events)
-dataset.count_f2f_consultation = selected_events.where(
-    clinical_events.snomedct_code.is_in(f2f_consultation)
-).count_for_patient()
-
-dataset.last_f2f_consultation_code = (
-    selected_events.where(clinical_events.snomedct_code.is_in(f2f_consultation))
-    .sort_by(clinical_events.date)
-    .last_for_patient()
-    .snomedct_code
-)
-
```

Comment:
because you're removing this, you're no longer using the `f2f_consultation` codelist, just checking that this what you want?